hoodie-client/src/main/java/com/uber/hoodie/io/HoodieAppendHandle.java

```diff
@@ -149,9 +149,10 @@ private void init(HoodieRecord record) {
   private Optional<IndexedRecord> getIndexedRecord(HoodieRecord<T> hoodieRecord) {
     Optional recordMetadata = hoodieRecord.getData().getMetadata();
     try {
-      Optional<IndexedRecord> avroRecord = hoodieRecord.getData().getInsertValue(schema);
-
+      Optional<IndexedRecord> avroRecord = hoodieRecord.getData().getInsertValue(originalSchema);
```

Comment:
why change to `originalSchema`? may be rebase on top of master again?

Reply:
> why change to `originalSchema`? may be rebase on top of master again?

yes, I rebased master code [originalSchema](https://github.com/apache/incubator-hudi/blob/master/hoodie-client/src/main/java/com/uber/hoodie/io/HoodieAppendHandle.java#L152)